src/abstractMatrix.js

```diff
@@ -924,6 +924,25 @@ export default function AbstractMatrix(superCtor) {
         }
 
         /**
+         * Returns the norm of a matrix.
+         * @param {string} type - "frob" (default) ou "max" return resp. the Frobenius norm and the max norm.
+         * @return {number}
+         */
+        norm(type = 'frob') {
+            var result = 0;
+            if (type === 'max') {
+                return this.max();
+            } else { // by default, norm = "frobenius"
+                for (var i = 0; i < this.rows; i++) {
+                    for (var j = 0; j < this.columns; j++) {
+                        result = result + Math.abs(this.get(i, j)) * Math.abs(this.get(i, j));
```

Comment:
You don't need to take the absolute value. The square value will always be positive.